ctms/schemas/addons.py

```diff
@@ -83,6 +83,14 @@ class Config:
 AddOnsInSchema = AddOnsBase
 
 
+class UpdatedAddOnsInSchema(AddOnsInSchema):
```

Comment:
Added new instead of into existing objects because I didnt want the timestamps as an option for api calls. Allowing this hidden model as a segway between api input and db input.

Reply:
`default_factory` is required over `default` for dynamic default values